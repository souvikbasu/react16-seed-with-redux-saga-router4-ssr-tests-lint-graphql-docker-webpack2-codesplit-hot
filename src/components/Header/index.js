import React from 'react';

import logo from './logo.svg';
import './style.css';

const Header = ({ user }) => (
  <header className="header">
    <img src={logo} className="logo" alt="logo" />
    <h1 className="title">Welcome to React {user ? user.firstName : ''}</h1>
  </header>
);

export default Header;
