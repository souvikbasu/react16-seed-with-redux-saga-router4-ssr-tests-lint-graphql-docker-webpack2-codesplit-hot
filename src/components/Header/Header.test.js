import React from "react";
import { shallow } from "enzyme";

import Header from "./";

describe("<Header />", () => {
  it("matches snapshot", () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
