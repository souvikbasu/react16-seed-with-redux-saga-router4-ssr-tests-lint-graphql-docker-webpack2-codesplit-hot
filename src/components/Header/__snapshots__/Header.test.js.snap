// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> matches snapshot 1`] = `
<header
  className="header"
>
  <img
    alt="logo"
    className="logo"
    src="logo.svg"
  />
  <h1
    className="title"
  >
    Welcome to React 
  </h1>
</header>
`;
